### DISCUSSION TITLE
Training stops with an error

### DISCUSSION THREAD

**Discussion by DarKxRaideR** [bmaltais/kohya_ss#193] [Q&A] [answered]
Hello, when I have set all parameters and hit "Train Model" I get the following error that I do not understand, maybe someone can advise:

===================================BUG REPORT===================================
Welcome to bitsandbytes. For bug reports, please submit your error trace to: https://github.com/TimDettmers/bitsandbytes/issues
For effortless bug reporting copy-paste your error into this form: https://docs.google.com/forms/d/e/1FAIpQLScPB8emS3Thkp66nvqwmjTEgxp8Y9ufuWTzFyr9kJ5AoI47dQ/viewform?usp=sf_link
================================================================================
CUDA SETUP: Loading binary D:\Windows10Benutzer\Documents\Kohya\kohya_ss\venv\lib\site-packages\bitsandbytes\libbitsandbytes_cuda116.dll...
use 8-bit Adam optimizer
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 1700
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 850
  num epochs / epoch数: 1
  batch size per device / バッチサイズ: 2
  total train batch size (with parallel & distributed & accumulation) / 総バッチサイズ（並列学習、勾配合計含む）: 2
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 100
Traceback (most recent call last):
  File "D:\Windows10Benutzer\Documents\Kohya\kohya_ss\train_network.py", line 573, in <module>
    train(args)
  File "D:\Windows10Benutzer\Documents\Kohya\kohya_ss\train_network.py", line 356, in train
    "ss_noise_offset": args.noise_offset,
AttributeError: 'Namespace' object has no attribute 'noise_offset'
Traceback (most recent call last):
  File "E:\DxR-StableDiffusion\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "E:\DxR-StableDiffusion\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "D:\Windows10Benutzer\Documents\Kohya\kohya_ss\venv\Scripts\accelerate.exe\__main__.py", line 7, in <module>
  File "D:\Windows10Benutzer\Documents\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\accelerate_cli.py", line 45, in main
    args.func(args)
  File "D:\Windows10Benutzer\Documents\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py", line 1104, in launch_command
    simple_launcher(args)
  File "D:\Windows10Benutzer\Documents\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py", line 567, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['D:\\Windows10Benutzer\\Documents\\Kohya\\kohya_ss\\venv\\Scripts\\python.exe', 'train_network.py', '--enable_bucket', '--pretrained_model_name_or_path=C:/Users/Marco/stable-diffusion-webui/models/Stable-diffusion/Custom-realisticVisionV13_v13-RAW Photo-.safetensors', '--train_data_dir=E:\\DxR-StableDiffusion\\MGTrainingPics\\LORA\\image', '--resolution=512,512', '--output_dir=E:\\DxR-StableDiffusion\\MGTrainingPics\\LORA\\model', '--logging_dir=E:\\DxR-StableDiffusion\\MGTrainingPics\\LORA\\log', '--network_alpha=1', '--save_model_as=safetensors', '--network_module=networks.lora', '--text_encoder_lr=5e-5', '--unet_lr=0.0001', '--network_dim=8', '--output_name=last', '--lr_scheduler_num_cycles=1', '--learning_rate=0.0001', '--lr_scheduler=cosine', '--lr_warmup_steps=10', '--train_batch_size=2', '--max_train_steps=100', '--save_every_n_epochs=1', '--mixed_precision=fp16', '--save_precision=fp16', '--seed=1234', '--cache_latents', '--bucket_reso_steps=64', '--xformers', '--use_8bit_adam', '--bucket_no_upscale']' returned non-zero exit status 1.

**Comment by reduxo1** [comment, marked as answer]
same issue here, it seems like many things have changed since this was uploaded. i dont expect this to be updated or answered, once people have it working for themselves they dont bother to update or change anything.

**Reply by quasiblob** [reply]
I get the same error. 

Reinstalled the GUI from start, followed every step. Same result still.

**Reply by reduxo1** [reply]
there is a fix here

https://github.com/bmaltais/kohya_ss/issues/192

you need to download the new training script and place it in the correct folder

**Reply by quasiblob** [reply]
@reduxo1 that seems to fix this one for me, thanks. @DarKxRaideR too should try this.

**Reply by DarKxRaideR** [reply]
> @reduxo1 that seems to fix this one for me, thanks. @DarKxRaideR too should try this.

Yeah it fixed my problem as well thank you so much for the help!

**Reply by quasiblob** [reply]
OK, mark the question answered so others will know this.
